src/app/components/toolbar/index.html

```diff
@@ -20,6 +20,13 @@
     >
     Specifies a section to group items within the toolbar. The section displays items in a separate horizontal row.
     </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="sky-toolbar-view-actions"
```

Comment:
This new section is a component not a property. The component will be placed along side your other toolbar items. An example of how this can be used is 


```
<sky-toolbar>
    <sky-toolbar-item>
      <button
        class="sky-btn sky-btn-default"
      >
        Button 1
      </button>
    </sky-toolbar-item>
    <sky-toolbar-item>
      <button
        class="sky-btn sky-btn-default"
      >
        Button 2
      </button>
    </sky-toolbar-item>
    <sky-toolbar-view-actions>
      <button
        class="sky-btn sky-btn-default"
        title="Sort descending"
        type="button"
      >
        <sky-icon
          icon="angle-double-down"
        >
        </sky-icon>
      </button>
      <button
        class="sky-btn sky-btn-default"
        title="Sort ascending"
        type="button"
      >
        <sky-icon
          icon="angle-double-up"
        >
        </sky-icon>
      </button>
    </sky-toolbar-view-actions>
  </sky-toolbar>
```

Reply:
The list version of this is not a property either. We'll need to fix it there too if that is how it is documented.

Reply:
I'll create a separate issue for updating the list version of the property